api/src/pcapi/core/offers/validation.py

```diff
@@ -72,6 +71,11 @@
     "minimum_score": [float, int],
 }
 
+OFFER_EXTRA_DATA_MANDATORY_FIELDS = [
+    "showType",
+    "musicType",
+]
```

Comment:
Déclarer cette constante directement comme un set évite d'avoir à faire la conversion à chaque utilisation dans `check_offer_extra_data()`.

```suggestion
OFFER_EXTRA_DATA_MANDATORY_FIELDS = {
    "showType",
    "musicType",
}
```